contrib/kind.sh

```diff
@@ -234,6 +234,32 @@ kubectl get secrets -o jsonpath='{.items[].data.token}' > /tmp/kind/token
 pushd ../go-controller
 make
 popd
+
+# Patch CoreDNS to work in Github CI
```

Comment:
should this patching only be done for a Github run and not in other users' deployments?

Reply:
yeah